packages/aws-cdk/lib/init-templates/app/typescript/package.template.json

```diff
@@ -6,6 +6,7 @@
   },
   "scripts": {
     "build": "tsc",
+    "clean": "tsc -b --clean",
```

Comment:
Ho, wait. I didn't notice this before, but I don't think it's a good idea to combine `-b` and non-`-b` builds. I remember having run into issues with mixing those previously (where some source didn't get rebuilt).

Will this work without the `-b`?

Reply:
Wow. Good catch, I didn't think about that. That would definitely causing problems in some projects. `--clean` is a flag on `--build` so it won't work without it. tsc really needs a clean flag on the compiler itself. 

The other options I can think of for this are less than ideal; maybe this needs to wait until there is a better option available.